starknet_py/net/client_models.py

```diff
@@ -103,7 +104,7 @@ class InvokeTransaction(Transaction):
 
     contract_address: int
     calldata: List[int]
-    entry_point_selector: int
+    entry_point_selector: Optional[int] = None
```

Comment:
This is a breaking change. Alternative solution is to use a default value of `0`, but I'm not sure if this is better.

Reply:
I don't think that defaulting to 0 is a good idea. I would leave it as optional @Solpatium

Reply:
The starknet model is changing and our job should be to model it properly. This field is not optional, so starknet.py should follow. Please add a comment when this is optional.